engine/src/srvcgi.cpp

```diff
@@ -707,12 +707,11 @@ static void cgi_store_form_urlencoded(MCVariable *p_variable, MCDataRef p_data,
 
 static void cgi_fix_path_variables()
 {
-	char *t_path, *t_path_end;
+    char *t_path;
 
 	MCStringRef env;
     env = nil;
-	t_path = nil;
-    t_path_end = nil;
+    t_path = nil;
     
     // SN-2014-07-29: [[ Bug 12865 ]] When a LiveCode CGI script has a .cgi extension and has
```

Comment:
Okay cool - as long as things work the same as they did before then that's fine.

Perhaps file a bug for later consideration about removing the engine-specific behavior if it is indeed fixable by web-server configuration.